rpcserver.go

```diff
@@ -913,6 +913,13 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 			activeNetParams.Params.Name)
 	}
 
+	// Check that we cannot send any coins to pubkeys.
```

Comment:
nit: comment might be better along the lines of "If the destination address parses to a valid pubkey, we assume the user accidentally tried to send funds to a bare pubkey address. Fail to prevent unintended transfers."

Reply:
that sounds better, will update

Reply:
@cfromknecht fixed!